src/main/java/com/onesignal/client/api/DefaultApi.java

```diff
@@ -158,7 +158,7 @@ public okhttp3.Call beginLiveActivityCall(String appId, String activityId, Begin
         Map<String, Object> localVarFormParams = new HashMap<String, Object>();
 
         // Adds client sdk version header
-        localVarHeaderParams.put("OS-Usage-Data", "kind=sdk, sdk-name=onesignal-java, version=1.2.2");
+        localVarHeaderParams.put("OS-Usage-Data", "kind=sdk, sdk-name=onesignal-java, version=2.1.0");
```

Comment:
With the change to use 1.3.0 as the release version, we just have to note that 2.1.0 java is 1.3.0

Reply:
Also if it looks ok, I updated the PR title

Reply:
I'm ok with this change because it's not user-impacting. Are you saying that we should either change this back or just internally note what it maps to?

Reply:
Since not user facing, we can just note or know internally of this nuance.